tensor_comprehensions/pybinds/pybind_engine.cc

```diff
@@ -59,13 +60,17 @@ PYBIND11_MODULE(tc, m) {
     std::cerr << "\n PyTorch installation is missing, binary will be useless \n"
               << e.what() << std::endl;
   }
-  py::class_<tc::ATenCompilationUnit>(m, "ATenCompilationUnit")
+  py::class_<tc::ATenCompilationUnit<tc::CudaTcExecutor>>(
+      m, "ATenCompilationUnit")
       .def(py::init<>())
-      .def("define", &tc::ATenCompilationUnit::define, "Define the TC language")
+      .def(
+          "define",
+          &tc::ATenCompilationUnit<tc::CudaTcExecutor>::define,
+          "Define the TC language")
       .def(
           "compile",
           [dlpack](
-              tc::ATenCompilationUnit& instance,
+              tc::ATenCompilationUnit<tc::CudaTcExecutor>& instance,
```

Comment:
can you create an alias something like:
```
using ATenCudaExecutor = tc::ATenCompilationUnit<tc::CudaTcExecutor> 
```
and use that everywhere in this file? Thanks.